docs/antora/modules/ROOT/pages/configuration/index.adoc

```diff
@@ -0,0 +1,1183 @@
+        SERVICE_MARKET_FIAT_CURRENCIES: 'EUR,USD,CHF,GBP,RUB',
+        SERVICE_MARKET_TARGET_PAIRS: 'LSK_BTC,LSK_EUR,LSK_USD,LSK_CHF,BTC_EUR,BTC_USD,BTC_CHF',
+        // EXCHANGERATESAPI_IO_API_KEY: ''
+      },
+    },
+    // Other types of configurations.
+  ],
+};
+----
+--
+=====
+
+=== Export
+
+[tabs]
+=====
+Properties::
++
+--
+[cols="2,1,1,3,3",options="header",stripes="hover"]
+|===
+|Property's Name
+|Datatype
+|Configurable
+|Description
+|Sample
+
+| `SERVICE_BROKER`
+| string
+| ✔
+| URL of the microservice message broker (NATS or Redis).
+| redis://localhost:6379/0
+ 
+| `SERVICE_EXPORT_REDIS`
+| string
+| ✔
+| URL of the permanent cache storage (Redis).
+| redis://localhost:6379/3
+
+| `SERVICE_EXPORT_REDIS_VOLATILE`
+| string
+| ✔
+| URL of the volatile cache storage (Redis).
+| redis://localhost:6379/4
+
+|===
+
+--
+Sample::
++
+--
+[source,js]
+----
+module.exports = {
+  apps: [
+    // Other types of configurations.
+    {
+      name: 'lisk-service-export',
+      // [...]
+      env: {
+        SERVICE_BROKER: 'redis://localhost:6379/0',
+        SERVICE_EXPORT_REDIS: 'redis://localhost:6379/3',
+        SERVICE_EXPORT_REDIS_VOLATILE: 'redis://localhost:6379/4',
+      },
+    },
+    // Other types of configurations.
+  ],
+};
+----
+--
+=====
+
+== Lisk Settings
+
+Configurable environment variables related to Lisk node.
+
+=== Node settings
+
+[cols="2,1,3,3",options="header",stripes="hover"]
+|===
+|Property's Name
+|Datatype
+|Description
+|Sample
+
+| `LISK_CORE_WS`
+| string
+| URL for Lisk Core WebSocket RPC API.
+The URL will differ according to the environment i.e., docker, or server installation.
+| ws://localhost:7887 or ws://host.docker.internal:7887
+ 
+| `LISK_CORE_CLIENT_TIMEOUT`
+| string
+| Lisk Core client timeout (in seconds).
+| 30
+|===
+
+
+=== MySQL Snapshot settings for Docker
+Configurations to sync Lisk Service from an existing snapshot and to speed up the syncing process in the Docker environment.
+
+[cols="2,1,3,3",options="header",stripes="hover"]
+|===
+|Property's Name
+|Datatype
+|Description
+|Sample
+
+| `ENABLE_APPLY_SNAPSHOT`
+| string
+| Enable or disable apply snapshot feature.
+| true
+ 
+| `INDEX_SNAPSHOT_URL`
+| string
+| Custom snapshot URL: expected to end with "sql.gz".
+| https://snapshots.lisk.io/mainnet/service.sql.gz
+
+| `DOCKER_COMPOSE_FILEPATH`
+| string
+| When MySQL is hosted as a docker-compose service, set the following environment variables.
+Set docker-compose file path by using the absolute path.
+| /Users/lisk/lisk-service/jenkins/mysql/docker-compose.yml
+
+| `DOCKER_MYSQL_SERVICE_NAME`
+| string
+| Set MySQL service name as defined in the above docker-compose file.
+| mysql
+
+|===
+
+
+
+
+== External services
+
+[cols="2,1,3,3",options="header",stripes="hover"]
+|===
+|Property's Name
+|Datatype
+|Description
+|Sample
+
+| `LISK_STATIC`
+| string
+| Lisk static assets, i.e., known account lists.
+| https://static-data.lisk.com
+ 
+| `GEOIP_JSON`
```

Comment:
Already mentioned within the connector. Do we need it here again?

Reply:
Can be removed!